configs/components/jsbach/jsbach.datasets.yaml

```
@@ -10,18 +13,44 @@ input_sources:
         "lctlib_foci": "${computer.pool_directories.focipool}/ECHAM6_GENERAL/JSBACH/input/lctlib_nlct21.def"
 
 forcing_sources:
-        "LU_1850": "${input_dir}/${echam.resolution}/New_Hampshire_LCC/hist_harvest/LUH_harvest_${echam.resolution}_1850.nc"
-        "LU_1950": "${input_dir}/${echam.resolution}/New_Hampshire_LCC/hist_harvest/LUH_harvest_${echam.resolution}_1950.nc"
+        # harvest
+        "LU_1850": "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${hist_fold}/LUH_harvest_${echam.resolution}_1850.nc"
+        "LU_1950": "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${hist_fold}/LUH_harvest_${echam.resolution}_1950.nc"
         "LU_HIST":
-                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/hist_harvest/LUH_harvest_${echam.resolution}_1850.nc":
+                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${hist_fold}/LUH_harvest_${echam.resolution}_1850.nc":
                         to: 1850
-                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/hist_harvest/LUH_harvest_${echam.resolution}_@YEAR@.nc":
+                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${hist_fold}/LUH_harvest_${echam.resolution}_@YEAR@.nc":
                         from: 1851
+                        to: 2014
+                # HIST scenario will use this data beyond year 2014
+                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${hist_scenario}_harvest/LUH_harvest_${echam.resolution}_@YEAR@.nc":
+                        from: 2015
```

```suggestion
                # HIST scenario will use this data beyond year 2014
                "${input_dir}/${echam.resolution}/New_Hampshire_LCC/${scenario}_harvest/LUH_harvest_${echam.resolution}_@YEAR@.nc":
                        from: 2015
```
@chrisdane , shouldn't this simply be the `jsbach.scenario`? The way is coded right now it will always load ssp370 one unless you specify in your runscript `jsbach.hist_scenario: ssp370` which is kind of a repetition if you already have the `general.scenario` defined already in the runscript

---

I dont have `jsbach.scenario`, you mean `jsbach.scenario_type` which is only `echam.scenario`? No, please leave it as it is as here a different thing than `echam.scenario` is needed. I think the variable name `hist_scenario` is not a good choice. Its rather confusing as there is only one historical scenario. The only purpose of the usage of this variable in the `LU_hist` definition is for years larger than 2014. This, in turn, only happens due to the necessity to load forcing data of the current year + 1 to do an interpolation. Hence, as the historical simulation in CMIP6 ends in 2014, the only data that is needed by echam is data from 2015 and we agreed to choose the one from the ssp370 scenario (it does not really matter which is chosen). Thus, I suggest to rename `hist_scenario` to something like  `hist_extend_scenario` and make a comment that explains its meaning right at its definition.

---

I see, thanks a lot. I'll rename the variable and copy part of your explanation as a comment on top of the variable.